posthog/models/event/sql.py

```diff
@@ -271,9 +259,11 @@
     events
 WHERE
     team_id = %(team_id)s AND
-    {property_field} ILIKE %(value)s
+    JSONHas(properties, %(key)s)
```

Comment:
Q: Could we rely on materialized columns here when possible?

Reply:
We were already doing so, but this check didn't need to be there for mat columns. Seems it was a bug that was also there before, fixed.